presentation/src/main/java/com/safetyheads/akademiaandroida/presentation/ui/activities/DashboardActivity.kt

```diff
@@ -27,6 +28,18 @@ class DashboardActivity : AppCompatActivity() {
         viewModel.isExistUser.observe(this) {
             setupNavBar(it)
         }
+
+        viewModel.isLogOut.observe(this) { isLogOut ->
+            if (isLogOut) {
+                val intent = Intent()
+                intent.setClassName(
+                    packageName,
+                    "com.safetyheads.akademiaandroida.presentation.ui.MainActivity"
```

Comment:
Why do we have to start activity by it's name? Can't we just do a regular intent with MainActivity.class?